documentation/release_notes.markdown

```diff
@@ -10,20 +10,46 @@ canonical: "/puppetserver/latest/release_notes.html"
 [puppetserver.conf]: ./config_file_puppetserver.markdown
 [product.conf]: ./config_file_product.markdown
 
-For release notes on versions of Puppet Server prior to Puppet Server 5, see [docs.puppet.com](https://docs.puppet.com/puppetserver/2.7/release_notes.html).
+For release notes on versions of Puppet Server prior to Puppet Server 5, see [docs.puppet.com](https://docs.puppet.com/puppetserver/2.8/release_notes.html).
 
 ## Puppet Server 5.1.0
 
 Released September 12, 2017
 
-This is a feature release of Puppet Server.
+This is a feature and bug-fix release of Puppet Server.
 
 ### New Feature: Automatic CRL refresh on certificate revocation
 
-Puppet Server 5.1.0 includes the ability to automatically refresh the CRL in the running SSL context when any changes to that file have occurred, namely the addition of a revoked certificate. Prior to this release, revoking an agent's certificate required restarting the Puppet Server process before that revocation would be honored and the agent denied authentication. Now revocation will be effective psuedo-immediately (some threshold of milliseconds) without requiring a restart of the server.
+Puppet Server 5.1.0 includes the ability to automatically refresh the certificate revocation list (CRL) in the running SSL context when any changes to that file have occurred, namely the addition of a revoked certificate. Prior to this release, revoking an agent's certificate required restarting the Puppet Server process before that revocation would be honored and the agent denied authentication. Revocation is now effective within milliseconds and does not require restarting server.
```

Comment:
This is a little off base since you didn't change exactly this bit, but while we're here (sorry to be that guy)...

1. "in the running SSL context" is an implementation detail, IMO. I think it's equally understandable to just say "refresh the Server's certificate revocation list (CRL) when any..."

2. "restarting" mentioned here should be "restarting or reloading", I think.